cli/src/commands/resolve.rs

```diff
@@ -130,9 +137,11 @@ pub(crate) fn print_conflicted_paths(
     conflicts: &[(RepoPathBuf, MergedTreeValue)],
     formatter: &mut dyn Formatter,
     workspace_command: &WorkspaceCommandHelper,
+    matcher: &dyn Matcher,
 ) -> Result<(), CommandError> {
     let formatted_paths = conflicts
         .iter()
+        .filter(|(path, _)| matcher.matches(path))
```

Comment:
you'll probably need to filter the input `conflicts` iterator. `formatted_paths` are zipped with `conflicts` later.

And I think it's still better to leave a TODO comment because `MergedTree::conflicts()` has a similar comment about matcher support.

Reply:
> you'll probably need to filter the input conflicts iterator. formatted_paths are zipped with conflicts later.

Good point... my code was buggy.

> And I think it's still better to leave a TODO comment because `MergedTree::conflicts()` has a similar comment about matcher support.

I see what you mean now. I've dropped my conflict-related changes and added the TODO.